### ISSUE TITLE
Some tasks don't appear in the `task` table

### ISSUE THREAD

**Issue by ahal** [klahnakoski/TestLog-ETL#65]
I'm trying to use ActiveData to analyze this push:
https://treeherder.mozilla.org/#/jobs?repo=try&revision=1876bd9a2696c69d2801306f13b238cd92df876e

I'm noticing that there are tasks that show up in the `treeherder` table, but don't show up in the `task` table (roughly 9 of them). Maybe it's possible they just haven't been processed yet and ActiveData will get to them soon, but the other tasks were available only a few hours after the push.

Here's a query that hits the `treeherder` table:
https://activedata.allizom.org/tools/query.html#query_id=DcPSnQQC

And here's one that should select the same thing on the `task` table:
https://activedata.allizom.org/tools/query.html#query_id=KBKrKvyd

The former returns a result, the latter doesn't.

**Comment by ahal** [comment]
Note, it looks like they aren't in the unittest table either.

**Comment by klahnakoski** [comment]
@ahal yes, the unittests are downstream from the tasks.  So, it makes sense they are missing

Some notes for self:

ETL block  https://active-data-task-cluster-logger.s3-us-west-2.amazonaws.com/tc.2756421%3A275641263.json.gz

query
```
{
	"format":"list",
	"from":"task",
	"limit":10000,
	"select":[{"name":"id","value":"task.id"}],
	"where":{"and":[{"eq":{"task.id":"b62nTxB2Sgae5PsNrSIXyg"}}]}
}
```

**Comment by klahnakoski** [comment]
This will ensure data is reprocessed sooner after a machine termination.

https://github.com/klahnakoski/ActiveData-ETL/commit/09e3dc63df99ec163457ecc1b7e51ee8e82a2f3a

I have not figured out why the block was not reprocessed
